setup.py

```diff
@@ -40,5 +41,5 @@
     ],
     package_dir={"": "cloudevents"},
     packages=["http", "sdk"],
-    version="1.0.0",
+    version=pypi_config["version_target"],
```

Comment:
I think I'd prefer getting these from a more generic location, like `cloudevents.__version__`.

See some examples here: https://packaging.python.org/guides/single-sourcing-package-version/

Reply:
working on it